.github/workflows/pr.yml

```diff
@@ -1,95 +1,195 @@
-name: PullRequest builder TLA+
+name: TLA+ PR Validation
 on: [pull_request]
 
 jobs:
-  build:
+  tlatools-build-and-test:
+    name: TLA+ Tools Build & Test
     runs-on: ubuntu-latest
-    env:
-      TLAPS_VERSION: 202210041448
-      MVN_COMMAND: xvfb-run mvn -Dtest.skip=true -Dmaven.test.failure.ignore=true
     steps:
-
-    - uses: actions/checkout@v2
+    - name: Clone tlaplus/tlaplus
+      uses: actions/checkout@v4
       with:
         # Number of commits to fetch. 0 indicates all history.
         # jgit task nested in customBuild.xml fails without history.
-        fetch-depth: '0'
-        
+        fetch-depth: 0
     - name: Set up JDK 11
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v4
       with:
+        distribution: adopt
         java-version: 11.0.3
-
-      ##
-      ## Run TLC tests.
-      ##
-    - name: Run TLC tests
+    - name: Build & Test Tools
       run: ant -f tlatools/org.lamport.tlatools/customBuild.xml -Dtest.halt=true compile compile-test test dist
-
-      ##
-      ## Build TLC and Toolbox (logger reduces verbosity).
-      ##
-    - name: Build with Maven (Linux)
-      run: $MVN_COMMAND -Dtycho.disableP2Mirrors=true -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -fae -B verify --file pom.xml
-
-      ##
-      ## Trigger build of CommunityModule as integration tests.
-      ##
-    - uses: actions/checkout@v2
+    - name: Clone tlaplus/CommunityModules
+      uses: actions/checkout@v4
       with:
         repository: tlaplus/CommunityModules
         path: communitymodules/
         # Number of commits to fetch. 0 indicates all history.
         # jgit task nested in customBuild.xml fails without history.
-        fetch-depth: '0'
-    - name: Build CommunityModules
+        fetch-depth: 0
+    - name: Build Community Modules as Integration Test
       run: |
-          mkdir -p communitymodules/tlc
-          cp tlatools/org.lamport.tlatools/dist/tla2tools.jar communitymodules/tlc/
-          ant -f communitymodules/build.xml -Dskip.download=true   
+        mkdir -p communitymodules/tlc
+        cp tlatools/org.lamport.tlatools/dist/tla2tools.jar communitymodules/tlc/
+        ant -f communitymodules/build.xml -Dskip.download=true   
 
-      ##
-      ## Trigger run of examples as integration tests.
-      ##
-    - uses: actions/checkout@v2
+  toolbox-build-and-test:
+    name: Eclipse Toolbox Build & Test
+    runs-on: ubuntu-latest
```

Comment:
Building (and testing) the Toolbox used to be possible on Win, Mac, and Linux prior to this PR.

Reply:
Continuing this conversation here: https://github.com/tlaplus/tlaplus/pull/900#issuecomment-2028845070